src/chapters/chapter1.md

```diff
@@ -1,19 +1,39 @@
-**Chapter Summary**
+# Chapter 1
 
-John Sr. disappears. Brothers come to town when Amy calls. Find the house a mess, but no sign of a crime, so cops won't investigate.
+The house looked just like he remembered it, though perhaps a little more worn. Even the tire swing was exactly where he'd left it, swinging gently in the autumn breeze. The leaves on the maple, the same one he and Patrick had hung that swing from when they were kids, when they were still best friends, had turned a brilliant red. How long had it been since he'd been home - or spoken to Patrick for that matter? Ten years at least. Probably more.
 
-father's disappearance was sudden and caught the children off guard - the brothers fly into town all and descend on their childhood home. Amy's already there looking through the mess that is their father's house. it's the first time they've seen one another in several years. she tells them what she knows - that dad's disappeared, the police won't investigate.
+He parked the rental car in the driveway, grabbed his overnight bag from the backseat and got out. With any luck, this would be a short trip. Really, it needed to be. He'd assured Alexa that he'd be back in a day or two - there had to be a explanation and Amy was probably overreacting, after all, he'd reasoned, if there was actually a problem, the police would be involved - and when he got home they would talk through everything.
 
-# Chapter 1
+Amy must have heard the tires spinning on the loose gravel because when he looked up she was standing in the doorway. Under different circumstances she would have smiled. Meanwhile, his failure to suppress a smile despite the circumstances only served as further evidence that she'd always been the most disciplined sibling. He couldn't help it. Hers was the first friendly face he'd seen in weeks. His shoulders, which he hadn't noticed he'd been clenching, relaxed at the sight of her. He dropped the duffel on the threshold and pulled her close, enveloping her slight frame in his massive arms, "How you doing, kid?"
```

Comment:
The section "He couldn't help it ... 'How you doing, kid?'" is reading with romantic overtones to me. It's certain phrases, all used together, maybe? e.g. "at the sign of her," "pulled her close," "enveloping her slight frame," and "his massive arms."